src/github/types/queries.ts

```diff
@@ -0,0 +1,7 @@
+export type GetContributorInfoQuery = {
```

Comment:
```suggestion
export interface GetContributorInfoQuery {
```

on préfère déclarer des interfaces que des types, c'est plus maniable